moonbeam-types-bundle/package.json

```diff
@@ -7,7 +7,7 @@
   "prepublish": "tsc",
   "scripts": {
     "compare": "rm -fr dist && tsc && npx ts-node check.ts",
-    "build": "tsc",
+    "build": "rm -fr dist && node_modules/.bin/tsc",
```

Comment:
Does this solve  particular problem? Same question in the CI.

Reply:
running tsc when there is already a dist folder doesn't work. Does it work for you?

Reply:
I don't think I've run into this problem. I just wanted to make sure. I trust you on TS stuff :)